mlrun/__main__.py

```diff
@@ -763,11 +763,12 @@ def get(kind, name, selector, namespace, uid, project, tag, db, extra_args):
             name, project=project, tag=tag, labels=selector
         )
         df = artifacts.to_df()[
-            ["tree", "key", "iter", "kind", "path", "hash", "updated", "uri"]
+            ["key", "iter", "kind", "path", "hash", "updated", "uri", "tree"]
         ]
         df["tree"] = df["tree"].apply(lambda x: f"..{x[-8:]}")
         df["hash"] = df["hash"].apply(lambda x: f"..{x[-6:]}")
         df["updated"] = df["updated"].apply(time_str)
+        df.rename(columns={"tree": "job/workflow uid"}, inplace=True)
```

Comment:
why rename and not change `df["tree"]` instead?

Reply:
so i will not change the use of "tree" everywhere in the code..

Reply:
i mean, why not
`df["job/workflow uid"] = df["tree"].apply(lambda x: f"..{x[-8:]}")`

Reply:
but then you will still have the "tree" column, which is not needed anymore